screens/ClerkLoginScreen.js

```diff
@@ -59,17 +62,34 @@ export default class ClerkLoginScreen extends React.Component {
           break;
         // TODO for production, we should have some sort of logging mechanism (i.e replacing console logs)
         case status.FOUND:
-          console.log('Incorrect password');
+          errMsg = 'Incorrect password';
           break;
         case status.NOT_FOUND:
-          console.log('No clerk found at this store');
+          errMsg = 'No clerk found at this store';
           break;
         case status.DUPLICATE:
-          console.log('Database malformed! Two users found');
+          errMsg = 'Database malformed! Two users found';
           break;
         default:
           return;
       }
+
+      if (lookupResult.errorMsg !== '') {
+        logAuthErrorToSentry({
+          screen: 'ClerkLoginScreen',
+          action: 'handleSubmit',
+          attemptedPin: '',
+          error: errMsg,
+        });
+        console.log(errMsg);
+      } else {
+        Sentry.configureScope((scope) => {
+          scope.setUser({
```

Comment:
The user logging isn't working `clerkRecord` is an object. You need to pull `clerkRecord.id` and also please add `username: clerkRecord.clerkName`